Pages/EnterSaveTransaction.js

```diff
@@ -0,0 +1,97 @@
+import { StatusBar } from 'expo-status-bar';
+import React from 'react';
+import { NavigationContainer } from '@react-navigation/native';
+
+import { Button, StyleSheet, Text, TextInput, View } from 'react-native';
+
+
+function EnterSaveTransaction({ navigation }) {
+    return (
+    <View style={styles.container}>
+         <Button 
+            style={styles.BackButton}
+            title="<--"
+            onPress={() => navigation.goBack()}
+        />
+        <Text>Enter Transaction in SAVE Bank:</Text>
+        <TextInput
+            placeholder="Amount: $"
+            style={styles.transactionAmount}
+        />
+        <TextInput
+            placeholder="Description"
+            style={styles.transactionDescription}
+        />
+      
+        <Button 
+            style={styles.Deposit}
+            title="Deposit Money"
+            //onPress={() => navigation.navigate('EnterShareTransaction')}
+            //i think this should not pop up to a new page but rather a notification
+            //kinda like venmo how it goes chaching for deposit and vibrate for withraw
+        />
+        <Button 
+            style={styles.Withdraw}
+            title="Withdraw Money"
+            //onPress={() => navigation.navigate('EnterShareTransaction')}
+            //i think this should not pop up to a new page but rather a notification
+            //kinda like venmo how it goes chaching for deposit and vibrate for withraw
+        />
+        </View>
+    )
+}
+
+const styles = StyleSheet.create({
```

Comment:
same style thing

Reply:
Just go over all of your style sheets and keep track of which things are the same. Take all of them out and put them in a theme class or something. DRY (Don't Repeat Yourself)